src/inspectors/ensureValidReceipt.js

```diff
@@ -8,7 +8,7 @@ export default function ensureValidReceipt (receipt) {
   let proofHash = receipt.targetHash;
   const merkleRoot = receipt.merkleRoot;
   try {
-    const proof = receipt.proof;
+    const proof = receipt.proof || receipt.path;
```

Comment:
And what if a V3 credential has the old proofing mechanism? That wouldn't be valid.

Reply:
what is the old proofing system in this case?

Reply:
MerkleProof2017. We wouldn't want a MerkleProof2017 receipt on a V3 credential, so if it makes it this far down, it wouldn't really be valid. But if that's handled somewhere else and this is just for the ease of use in processing the receipt then that's okay - I just wanted to bring up this edge case.

Reply:
ah right, yeah we can have a safety check somewhere. I'll add that to this PR.